### PR TITLE
Fix: Expansion icons appear twice on corporation cards

### PR DESCRIPTION
**Pull request by nwai90**
Screenshot 2020-10-25 at 3 05 13 PM

### CODE REVIEW

File: src/styles/cards.less [hunk 1]
```diff
@@ -1493,6 +1493,13 @@ input[type="radio"]:checked + .filterDiv::after {
     text-align:center;
 }
 
+.card-utopia-invest {
```
Comment (kberg) [inline src/styles/cards.less:1496]: Is this related to expansion icons? If not, is it intentional?  Just asking for you to remove it if it was accidental.

Comment (kberg) [reply src/styles/cards.less:1496]: Wait, sorry I can read.

Comment (nwai90) [reply src/styles/cards.less:1496]: @kberg Intentional. The current layout is broken

Screenshot 2020-10-25 at 10 46 30 PM

**Review by kberg — APPROVED** [review]
